### PR TITLE
adding pages for signup

### PR DESCRIPTION
**Pull request by marinaRomanchuk**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
+                        <Route path="/signup" element={<SignUp />} />
+                        <Route path="/search" element={<Search />} />
+                    </Routes>
+            </div>
+        </BrowserRouter>
+    );
 }
 
-export default App;
+export default App;
```
Comment (revimi) [inline src/App.js]: Please add a newline here and in some other files

File: src/components/SignIn.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        e.preventDefault();
+        console.log(`Form submitted`);
+    }
+
+    onSigninClick = () => {
+        const userData = {
+            username: this.state.username,
+            password: this.state.password
+        };
+        axios.post(`http://127.0.0.1:8000/api/token/`, userData)
```
Comment (revimi) [inline src/components/SignIn.js]: Please don't hardcode URLs. The staging and prod will have another addresses

File: src/components/SignUp.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+            repeatedPassword: ''
+        }
+    }
+
+    onSignupClick = () => {
+        const userData = {
+            username: this.state.username,
+            password: this.state.password
+        };
+        axios.post(`http://127.0.0.1:8000/api/signup/`, userData)
```
Comment (revimi) [inline src/components/SignUp.js]: Same here

**Review by revimi — COMMENTED** [review]
Looks good. left few comments

File: src/App.js (hunk 1 above)
Comment (marinaRomanchuk) [reply src/App.js]: Done

File: src/components/SignIn.js (hunk 2 above)
Comment (marinaRomanchuk) [reply src/components/SignIn.js]: I use just `api/token/ ` in this case now

File: src/components/SignUp.js (hunk 3 above)
Comment (marinaRomanchuk) [reply src/components/SignUp.js]: I use just `api/signup/` in this case now

File: src/components/SignIn.js (hunk 2 above)
Comment (revimi) [reply src/components/SignIn.js]: And how does the browser know where to go? 
For example, the app domain may be `app.instagram.com` and the API domain `api.instagram.com` 
Let's have some common place to store variables like "the API path"

File: src/components/SignUp.js (hunk 3 above)
Comment (revimi) [reply src/components/SignUp.js]: Same comment as above

File: src/components/SignIn.js (hunk 2 above)
Comment (marinaRomanchuk) [reply src/components/SignIn.js]: This place is PathList.js now

File: src/components/SignUp.js (hunk 3 above)
Comment (marinaRomanchuk) [reply src/components/SignUp.js]: This place is PathList.js now

File: src/components/PathList.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+export const ROOT_API = "http://127.0.0.1:8000";
+export const SIGNUP_API = "http://127.0.0.1:8000/api/signup/";
+export const TOKEN_API = "http://127.0.0.1:8000/api/token/";
+
```
Comment (revimi) [inline src/components/PathList.js]:
```suggestion
export const API_DOMAIN = "127.0.0.1:8000";
```

Comment (revimi) [reply src/components/PathList.js]: Let's do smth like that. In the future, we'll have different domains for different stages

**Review by revimi — CHANGES_REQUESTED** [review]

File: src/components/PathList.js [hunk 5]
```diff
@@ -0,0 +1 @@
+export const API_DOMAIN = "http://127.0.0.1:8000";
```
Comment (revimi) [inline src/components/PathList.js]: Domain does not include the `http` prefix
```suggestion
export const API_DOMAIN = "127.0.0.1:8000";
```

**Review by revimi — APPROVED** [review]

File: src/components/PathList.js (hunk 5 above)
Comment (marinaRomanchuk) [reply src/components/PathList.js]: Done

**Pull request merged**